conda/plugins/windows.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+
+import platform
+
+from .. import plugins
+
+
+@plugins.register
+def conda_virtual_packages():
+    if platform.system() != "Windows":
+        return
+
+    yield plugins.CondaVirtualPackage("win", None)
```

Comment:
I understand this replicates current functionality, but why don't we include the version here?